keras_hub/src/models/whisper/whisper_audio_converter.py

```diff
@@ -6,8 +6,10 @@
 
 try:
     import tensorflow as tf
+    import keras.ops as ops
```

Comment:
No reason to import keras conditionally. We don't want to assume tf is install, we can assume keras is installed.